src/api.js

```diff
@@ -10,7 +10,7 @@ function getAllMovies() {
 }
 
 function getMovie(id) {
-  return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+  return fetch(`https://rancid-tomatillos.herokuapp.com/ai/v2/movies/${id}`)
```

Comment:
forgot to change back the valid url from when testing 500 error

Reply:
You can easily test 500 errors with Cypress 😉 - No need to change the URL in your React code.